### PR TITLE
[NTH-99] feat: patch therapist to activate account

### PR DESCRIPTION
**Pull request by ahmedmurtaja**
- Create an API to Patch Therapist activity for false / true
- Route should be protected for admin only
- After activating a therapist, he should receive an email with welcoming message as well as his profile link

**Validation:**

- active=true/false (required field)
- therapistId as number (required field)

### CODE REVIEW

**Review by YasserAmro0 — APPROVED** [review]

**Review by raghdarizq — APPROVED** [review]

File: src/controllers/admin.ts [hunk 1]
```diff
@@ -70,4 +73,54 @@ const getTherapistsForAdmin = async (req: Request, res: Response, next: NextFunc
...
+    // eslint-disable-next-line no-restricted-globals
+    if (isNaN(therapistId)) {
+      throw templateErrors.BAD_REQUEST('Therapist id should be a valid number');
+    }
+
+    if (active !== 'true' && active !== 'false') {
+      throw templateErrors.BAD_REQUEST('Active should be a valid boolean');
+    }
+
+    const therapist = await patchTherapist(therapistId, active === 'true');
```
Comment (LinaYahya) [inline src/controllers/admin.ts:90]: what if active is undefined?
So the condition to check active undefined is meaningless somehow

File: src/routes/admin.ts [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
-import { adminLogin, getTherapistsForAdmin } from '../controllers';
+import { adminLogin, getTherapistsForAdmin, updateTherapistActive } from '../controllers';
 import { checkAuth } from '../middlewares';
 import { RolesForSelect } from '../types';
 
 const router = Router();
 
 router.post('/login', adminLogin);
 router.get('/therapists', checkAuth(RolesForSelect.admin), getTherapistsForAdmin);
+router.patch('/therapists/:id/:active', checkAuth(RolesForSelect.admin), updateTherapistActive);
```
Comment (LinaYahya) [inline src/routes/admin.ts:10]: not sure why active is a request params, please send it in request body, and maybe we can add therapistId as well

**Review by LinaYahya — DISMISSED** [review]

File: src/controllers/admin.ts (hunk 1 above)
Comment (ahmedmurtaja) [reply src/controllers/admin.ts:90]: I didn't get  it, if active undefined it should throw a BAD_REQUEST

**Review by MohannadSabaa — APPROVED** [review]

File: src/config/index.ts [hunk 3]
```diff
@@ -9,7 +9,7 @@ const {
   TOKEN_TYPE, EXPIRY_DATE, REDIRECT_URIS, PROJECT_ID, AUTH_URI, TOKEN_URI,
   AUTH_PROVIDER_X509_CERT_URL, SECRET_KEY, TOKEN_TEST_THERAPIST, API_KEY, MAILERUSER, MAILERPASS,
   AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, REGION,
```
Comment (ahmedmurtaja) [inline src/config/index.ts:11]: vvv

**Pull request merged**